benchmark/matmul_metal/benchmark_matmul_metal.py

```
@@ -91,26 +217,35 @@ def bench_tilelang(M, N, K, block_M, block_N, block_K, warmup, repeats):
+            mlx_ratio = tl / mlx_tflops * 100 if mlx_tflops else None
             if tl > best_tflops:
                 best_tflops = tl
-                best_config = (bM, bN, bK)
-            print(f"{f'({bM},{bN},{bK})':>16s} | {tl:>10.1f} TFLOPS | {ratio:>5.0f}%")
+                best_config = (mode, bM, bN, bK, threads, swizzle_panel, swizzle_order)
+            mlx_text = f"{mlx_ratio:>7.0f}%" if mlx_ratio is not None else "     N/A"
+            print(
+                f"{mode:>10s} | {block_text:>16s} | {threads:>4d} | {swizzle_text:>8s} | "
+                f"{tl:>10.1f} TFLOPS | {torch_ratio:>7.0f}% | {mlx_text}"
+            )
         except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)